packages/flutter/test/material/icon_button_test.dart

```diff
@@ -3032,6 +3032,62 @@ void main() {
         ..rect(color: const Color(0xFF00FF00)), // IconButton overlay.
     );
   });
+
+  testWidgets('IconButton hovered & onLongPressed M2 - M3', (WidgetTester tester) async {
```

Comment:
Can we split this test into two?  Please make sure the title start 'Material2 - xx' and `Material3 - xx'.  

Combining the checks into a loop would make it difficult to read and update in future.